sources/lib/api/setup.py

```diff
@@ -29,7 +29,6 @@
               'gcp.data',
               'gcp.stackdriver',
```

Comment:
We'll need to have this be consistent with other things like BQ. Can you pick one -- stackdriver or monitoring and put everything in there rather than in nested modules?